ci_framework/roles/build_containers/molecule/hotfix/converge.yml

```diff
@@ -0,0 +1,56 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+
+- name: Converge
+  hosts: all
+  vars:
+    cifmw_build_containers_cleanup: true
+    cifmw_build_containers_config_file: "{{ ansible_user_dir }}/src/github.com/openstack-k8s-operators/ci-framework/roles/build_containers/files/containers.yaml"
+    cifmw_build_containers_hotfix_images: quay.io/podified-antelope-centos9/openstack-keystone:current-podified
```

Comment:
not sure we should hardcode that release. Would it be possible to consume "master" at this point?

Reply:
no issue with me

Reply:
fair enough. Just something to consider once antelop is EOL or the like.